engineering/conventions-go.md

```diff
@@ -0,0 +1,50 @@
+
+# Go Development Conventions
+
+## Introduction
+
+This guide documents development conventions for Go at source{d}. Check [general conventions](conventions.md) for language-independent conventions that are also applicable for Go projects.
+
+## Supported Go Versions
+
+* Support latest two stable major versions of Go (e.g. 1.9.x and 1.10.x).
+  * Some of our projects may support more versions, specially when there is wide-adoption outside source{d} (e.g. [go-git](https://github.com/src-d/go-git) currently supports three).
+
+## Dependency Management
+
+* If your project is an application:
+  * Use [dep](https://github.com/golang/dep) to manage dependencies.
+  * Check in the `vendor` directory into git.
+* If your project is a library:
```

Comment:
We also use `gopkg.in` for versioning libraries.

Reply:
@dennwc Do we still use it for new projects?

Reply:
of course, why not? and more with go `dep` battle in the core.

Reply:
Hope `vgo` will be accepted soon - it should solve few problems of `gopkg.in` at least.